libs/blocks/merch-card/merch-card.js

```diff
@@ -228,6 +228,24 @@ const simplifyHrs = (el) => {
   });
 };
 
+function createQuantitySelect(el) {
+  const quantitySelectConfig = el.querySelector('ul');
+  if (!quantitySelectConfig) return null;
+  const configMarkup = quantitySelectConfig.querySelector('li');
+  if (!configMarkup || !configMarkup.textContent.includes('Quantity')) return null;
+  const config = configMarkup.querySelector('ul').querySelectorAll('li');
+  if (config.length !== 2) return null;
+  const attributes = {};
+  attributes.title = config[0].textContent.trim();
+  const quantityValues = config[1].textContent.split(',').map((value) => value.trim()).filter((value) => /^\d+$/.test(value));
```

Comment:
100 max char per line please

Reply:
As per Milo guidelines the limit is 100 characters. It would be good if we can keep new lines under that limit and later we can clean up the rest.

Reply:
Changed.